src/views/Conversion.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+  <section class="pb-10" id="conversion-view">
+    <!-- Business Information page-->
+    <v-slide-x-transition hide-on-leave>
+      <div v-if="!isSummaryMode || !showFeeSummary">
+        <header>
+          <h1>Business Information</h1>
```

Comment:
I think "Business Information" should be "Record Conversion" according to the new [wireframes design] (https://preview.uxpin.com/363a49b33bdb2a314ccac0f18daad8969e93b337#/pages/149273920/simulate/sitemap).

Reply:
Fixed in next commit.